Gemfile

```diff
@@ -35,7 +35,7 @@ gem 'ruby-progressbar'
 group :development, :test do
   gem 'mocha'
   gem 'sqlite3'
-  gem 'rspec-rails'
+  gem 'rspec-rails', '4.0.0.beta2'
```

Comment:
:wave: rspec-rails 4 was released.  Maybe it's possible to drop the version specification?